01/userinfamous/wordvalue.py

```diff
@@ -1,33 +1,30 @@
+#Imports and appending path to reach beyond parent dir
+import os, sys
+sys.path.append('..')
 from data import LETTER_SCORES, DICTIONARY 
-import os
+
 
 #File path for Dictionary
 fname = os.path.join('../',DICTIONARY)
 
 #Wordlist from dictionary
 def load_words() -> list:
     with open(fname,'r') as f:
-        wordlist = f.read().split()
-        return wordlist
+        wordlist = f.read().split('\n')
+    return wordlist[:-1]
```

Comment:
why the slice? Because the last element is empty? This is not obvious for a reder and seems strange